app/controllers/application_controller.rb

```diff
@@ -76,7 +76,7 @@ def sort_events(events, sort)
     if sort == 'summary'
       [[Time.now, events.sort_by_summary]]
     else
-      events.sort_by_time.group_by_day(&:dtstart)
+      events.distinct.sort_by_time.group_by_day(&:dtstart)
```

Comment:
(non blocking) at some point we need to clean a lot of this type of thing out of the application controller